apps/worker/src/worker.spec.ts

```
@@ -108,4 +112,74 @@ describe("worker", () => {
+		test("should disable auto top-up after 7 days of payment failures", async () => {
+			await db.insert(tables.organization).values({
+				id: "org-disable-auto-topup",
+				name: "Disable Auto Top-up",
+				billingEmail: "billing@example.com",
+				credits: "0",
+				autoTopUpEnabled: true,
+				autoTopUpThreshold: "10",
+				autoTopUpAmount: "10",
+				paymentFailureCount: 8,
+				lastPaymentFailureAt: new Date(),
+				paymentFailureStartedAt: new Date(Date.now() - 8 * 24 * 60 * 60 * 1000),
```

ESLint

[error] Unexpected mix of '-' and '*'. Use parentheses to clarify the intended order of operations.

(no-mixed-operators)